lib/grover/js/processor.js

```diff
@@ -124,6 +124,18 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       page.setGeolocation(geolocation);
     }
 
+    const failOnRequest = options.requestFailure; delete options.requestFailure;
+    if (failOnRequest) {
+      page.on('requestfinished', (request) => {
+        if (request.response() && !request.response.ok()) {
```

Comment:
hmm looks like this should be `!request.response().ok()`  ?

```suggestion
        if (request.response() && !request.response().ok()) {
```

Reply:
You are correct.